src/plugins/multiline/verticalify.ts

```diff
@@ -109,10 +115,11 @@ export function unverticalify(elem: Element, force?: boolean) {
       delete child.dataset.isMultiline;
 
       // revert linebreaks to original symbol to get rid of <br>
-      if (child.dataset.isLineBreak) {
+      if (child.dataset.isLineBreak ?? child.dataset.isManualLineBreak) {
```

Comment:
Might be worth renaming `isLineBreak` to `isAutoLineBreak` just to be clear about the distinction.

Reply:
Agreed.